<launch>
	<node pkg="adeye" name="point_cloud_converter" type="point_cloud_converter">
		<remap from="points2_in" to="velodyne_points"/>
		<remap from="points_out" to="velodyne_points_converted" />
	</node>
</launch>
